conf/modules.config

```diff
@@ -685,9 +685,8 @@ process {
         ext.prefix = { "${meta.id}" }
         publishDir = [path: { "${params.outdir}/Annotation/MetaEuk/${meta.assembler}/${meta.id}" }, mode: params.publish_dir_mode, saveAs: { filename -> filename.equals('versions.yml') ? null : filename }]
     }
-
-    withName: MULTIQC {
-        ext.args   = { params.multiqc_title ? "--title \"${params.multiqc_title}\"" : '' }
+    withName: 'MULTIQC' {
```

Comment:
Umm, I thought I had read somewhere from Ben that quotes are not necessary for simple process selectors and that we agreed not to quote anymore 🤔